data/spells/scripts/party/enchant.lua

```diff
@@ -10,9 +10,9 @@ condition:setParameter(CONDITION_PARAM_STAT_MAGICPOINTS, 1)
 condition:setParameter(CONDITION_PARAM_BUFF_SPELL, true)
 
 local baseMana = 120
```

Comment:
You can actually move this var inside the function, it's not used anywhere else.